lib/membrane_rtc_engine/engine.ex

```diff
@@ -138,39 +138,13 @@ defmodule Membrane.RTC.Engine do
 
   @registry_name Membrane.RTC.Engine.Registry.Dispatcher
 
-  @type stun_server_t() :: ExLibnice.stun_server()
-  @type turn_server_t() :: ExLibnice.relay_info()
-
-  @typedoc """
-  SFU network configuration options.
-
-  `dtls_pkey` and `dtls_cert` can be used e.g. when there are a lot of SFU instances
-  and all of them need to use the same certificate and key.
-
-  Example configuration can look like this:
-
-  ```elixir
-  network_options: [
-    stun_servers: [
-      %{server_addr: "stun.l.google.com", server_port: 19_302}
-    ]
-  ]
-  """
-  @type network_options_t() :: [
-          stun_servers: [stun_server_t()],
-          turn_servers: [turn_server_t()],
-          dtls_pkey: binary(),
-          dtls_cert: binary()
-        ]
-
   @typedoc """
   SFU configuration options.
```

Comment:
```suggestion
  RTC Engine configuration options.
```

Reply:
I see that docs for `Membrane.RTC.Engine` are outdated. We can leave it as it is for now and fix docs for the whole module before merging.